content/dictionary_pages/letter_1.md

```diff
@@ -0,0 +1,35 @@
+title: А
+slug: letter-1
```

Comment:
I don't find it natural to call them like this. What about [scientific transliteration of Cyrillic](https://ru.wikipedia.org/wiki/%D0%9D%D0%B0%D1%83%D1%87%D0%BD%D0%B0%D1%8F_%D1%82%D1%80%D0%B0%D0%BD%D1%81%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8F_%D0%BA%D0%B8%D1%80%D0%B8%D0%BB%D0%BB%D0%B8%D1%86%D1%8B)?